TTS/bin/eval_encoder.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import torch
+from argparse import RawTextHelpFormatter
+
+from tqdm import tqdm
+
+from TTS.config import load_config
+from TTS.tts.datasets import load_tts_samples
+from TTS.tts.utils.speakers import SpeakerManager
+
+parser = argparse.ArgumentParser(
```

Comment:
It would be better to make this a function so that people can call it from outside

Reply:
Done